### ISSUE TITLE
Lonely <br> tags in payment.html template

### ISSUE THREAD

**Issue by dudzio12** [magento/magento2#33820]
### Summary (*)
When checkout reCaptcha is not visible or visible and configured to use invisible reCaptcha, there are two lonely `<br>` tags inside `payment.html` file:
https://github.com/magento/magento2/blob/2.4-develop/app/code/Magento/Checkout/view/frontend/web/template/payment.html

### Examples (*)

```
                </legend><br />
                <!-- ko foreach: getRegion('place-order-captcha') -->
                    <!-- ko template: getTemplate() --><!-- /ko -->
                <!-- /ko -->
                <br />
```

### Proposed solution

I suggest to remove them completely or move them to reCaptcha templates.

```
                </legend>
                <!-- ko foreach: getRegion('place-order-captcha') -->
                    <!-- ko template: getTemplate() --><!-- /ko -->
                <!-- /ko -->
```

---

Please leave me a feedback on what is the desired solution, so I can prepare PR for this.

- [x] Severity: **S4** _- Affects aesthetics, professional look and feel, “quality” or “usability”._

**Comment by mrtuvn** [comment]
It's self closing tag but must be a reason we have that tag in that template. I assume they want add new line for the captcha. But it's still valid html

**Comment by dudzio12** [comment]
@mrtuvn There are the other ways to achieve that html-less. With two `<br>` we end up with 2 line breaks, when captcha is disabled or invisible. That's why I think it should be removed completely and moved to stylesheet shoulders, which will implement basic margin on captcha element itself, only when `place-order-captcha` region is not empty.

**Comment by vertic4l** [comment]
using <br> tags to create space is such a bad idea

**Comment by engcom-Echo** [comment]
According to MDN Web docs, Creating separate paragraphs of text using <br> is not only bad practice, it is problematic for people who navigate with the aid of screen reading technology. Screen readers may announce the presence of the element, but not any content contained within <br>s. This can be a confusing and frustrating experience for the person using the screen reader. Hence we can consider removing the same.

**Comment by magepro-dev** [comment]
@magento I am working on this

**Comment by magepro-dev** [comment]
@magento add to contributors team

**Comment by magepro-dev** [comment]
@magento I am working on this

**Comment by dudzio12** [comment]
It's already fixed by #33892.
